federated_content_connector/course_metadata_importer.py

```diff
@@ -0,0 +1,192 @@
+"""Course metadata importer."""
+
+import logging
+from urllib.parse import quote_plus
+
+from common.djangoapps.course_modes.models import CourseMode
+from django.contrib.auth import get_user_model
+from openedx.core.djangoapps.catalog.models import CatalogIntegration
+from openedx.core.djangoapps.catalog.utils import get_catalog_api_base_url, get_catalog_api_client
+from openedx.core.djangoapps.content.course_overviews.models import CourseOverview
+
+from federated_content_connector.models import CourseDetails
+
+EXEC_ED_COURSE_TYPE = "executive-education-2u"
```

Comment:
Should this be a setting in edx-internal?

Reply:
I think we can place whatever we want in plugin.